packages/frontend/src/pages/Executions/index.tsx

```diff
@@ -49,18 +88,38 @@ export default function Executions(): React.ReactElement {
   return (
     <Box sx={{ py: 3 }}>
       <Container variant="page">
-        <Grid container sx={{ mb: [0, 3] }} columnSpacing={1.5} rowSpacing={3}>
-          <Grid
-            container
-            item
-            xs
-            sm
-            alignItems="center"
-            order={{ xs: 0, height: 80 }}
-          >
-            <PageTitle title={EXECUTIONS_TITLE} />
-          </Grid>
-        </Grid>
+        <Flex
+          flexDir={{ base: 'column', sm: 'row' }}
+          justifyContent="space-between"
+          alignItems="center"
+          gap={4}
+          mb={8}
+        >
+          <PageTitle title={EXECUTIONS_TITLE} />
+          <InputGroup w="25rem">
+            <InputLeftElement>
+              <Icon as={BiSearch} boxSize={5} />
+            </InputLeftElement>
+            <Input
+              textStyle="body-1"
+              pr="7.5rem"
```

Comment:
<img width="555" alt="Screenshot 2023-10-06 at 11 08 11 PM" src="https://github.com/opengovsg/plumber/assets/10072985/505ce593-a89a-4df7-a1aa-0da7373203e4">

the hardcoded padding will cause this slight overflow when button expands in width for 'Pending' state. Need to hardcode filter button width as well.

Reply:
i think hardcoding is a bit wonky here cus the button padding on the left and right will not be uniform, decided to add a resizeObserver to track the filter element and adjust the padding